### PR TITLE
Fix documentation  INITIAL_ADMIN_* variables

### LINKED ISSUE #2092: Fix INITIAL_ACCOUNT environment variable documentation master
## Before you open your issue
- [X] Check if no issue or pull-request for this already exists.
- [X] Check [documentation](https://mailu.io/master/) and [FAQ](https://mailu.io/master/faq.html). (Tip, use the search function on the documentation page)
- [X] You understand `Mailu` is made by volunteers in their **free time** — be conscise, civil and accept that delays can occur.
- [X] The title of the issue should be short and simple. It should contain specific terms related to the actual issue. Be specific while writing the title.

### Environment
 - [X] docker-compose

### Versions
To find your version, get the image name of a mailu container and read  the version from the tag (example for version 1.7).

```
master e7f7787 Wed Dec 15 2021
```

## Description

**Issue 1**: https://mailu.io/1.8/compose/setup.html?highlight=initial_account describes:

> You can have the system create it automatically: use the environment variables INITIAL_ACCOUNT* 

This environment variable does not exist and should be `INITIAL_ADMIN_ACCOUNT`.  
A suggestion is to refer to [Admin account - automatic creation](https://mailu.io/1.8/configuration.html#admin-account) only.

**Issue 2**: https://mailu.io/1.8/configuration.html#admin-account

```
INITIAL_ADMIN_ACCOUNT = ``root`` The first part of the e-mail address (ROOT@example.com)
INITIAL_ADMIN_DOMAIN = ``example.com`` the domain appendix. Most probably identical to the DOMAIN variable
INITIAL_ADMIN_PW = ``password`` the chosen password for the user
```

Proposal to remove spaces around the `=` and remove back-quotes.

## Replication Steps

Add the following lines to `mailu.env` and recreate `admin` container:

```bash
INITIAL_ADMIN_MODE=ifmissing
INITIAL_ADMIN_ACCOUNT=admin
INITIAL_ADMIN_DOMAIN=domain.nl
INITIAL_ADMIN_PW=mysecretpassword
```

Proposal is to document the 4 environment variables above in one example at a single place.

## Expected behaviour
Create admin account automatically by configuring environment variables after updating documentation. 

This is a nice and useful feature. Thanks!

### PR DESCRIPTION
**Pull request by Erriez**
## What type of PR?

Fix `master` documentation `INITIAL_ADMIN_*` environment variables:
- `setup.rst`
- `configuration.rst`

## What does this PR do?

Fix documentation `Docker Compose setup` and `Web settings | Admin account`.

### Related issue(s)
- Mention an issue like: #2092
- Auto close an issue like: closes #2092

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [X] In case of feature or enhancement: documentation updated accordingly
- [ ] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
...
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: one of the options below how the code should behave when it will try to create the admin user:
+  
+  - ``create``: (default) Will try to create user and will raise an exception if present.
+  - ``ifmissing``: if user exists, nothing happens, else it will be created.
+  - ``update``: user is created or, if it exists, its password gets updated.
+
+You should set ``INITIAL_ADMIN_MODE`` also to either ``update`` or ``ifmissing``. Leaving it with the default value could cause errors when restarting the system.
```
Comment (Diman0) [inline docs/configuration.rst:171]: Bad wording `also to either`. I recommend to completely change line 171 to:
It is recommended to set ``INITIAL_ADMIN_MODE`` to either ``update`` or ``ifmissing``. Leaving it with the default value will cause an error when the system is restarted.

File: docs/configuration.rst [hunk 2]
```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
 
 To have the account created automatically, you just need to define a few environment variables:
 
-.. code-block:: bash
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: one of the options below how the code should behave when it will try to create the admin user:
+  
+  - ``create``: (default) Will try to create user and will raise an exception if present.
```
Comment (Diman0) [inline docs/configuration.rst:167]: will try to create `an` user and will raise an exception if `the admin account already exists`.
All other lines start lowercase. The first word 'will' should be lowercase.

File: docs/configuration.rst [hunk 3]
```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
...
+  - ``ifmissing``: if user exists, nothing happens, else it will be created.
```
Comment (Diman0) [inline docs/configuration.rst:168]: if `the` user exists,

File: docs/configuration.rst [hunk 4]
```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
 
 To have the account created automatically, you just need to define a few environment variables:
 
-.. code-block:: bash
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: one of the options below how the code should behave when it will try to create the admin user:
```
Comment (Diman0) [inline docs/configuration.rst:165]: strange wording. Maybe change it to
`use one of the options below for configuring how the admin account must be created`

**Review by Diman0 — CHANGES_REQUESTED** [review]
It is much better than before. There are some small things I recommended to change to improve the readability.

File: docs/configuration.rst (hunk 1 above)
Comment (Erriez) [reply docs/configuration.rst]: Thanks for your feedback. I've improved this in an updated commit.

File: docs/configuration.rst (hunk 2 above)
Comment (Erriez) [reply docs/configuration.rst]: Updated (reworded a little bit).

File: docs/configuration.rst (hunk 3 above)
Comment (Erriez) [reply docs/configuration.rst]: Updated (reworded a little bit).

File: docs/configuration.rst (hunk 4 above)
Comment (Erriez) [reply docs/configuration.rst]: Updated.

File: docs/configuration.rst [hunk 5]
```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
 
 To have the account created automatically, you just need to define a few environment variables:
 
-.. code-block:: bash
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: use one of the options below for configuring how the admin account must be created:
+  
+  - ``create``: (default) creates a new admin account and raises an exception when already exists.
```
Comment (Diman0) [inline docs/configuration.rst:167]: (default) creates a new admin account and raises an exception when `it` already exists.

**Review by Diman0 — CHANGES_REQUESTED** [review]
Only one small typo. Everything else looks fine now.

File: docs/configuration.rst (hunk 5 above)
Comment (Erriez) [reply docs/configuration.rst]: Fixed. Thanks!

**Review by Diman0 — APPROVED** [review]
Lgtm. Thank you for your contribution.

**Review by nextgens — APPROVED** [review]

**Pull request merged**